### DISCUSSION TITLE
about C++ input output and powershell7 download

### DISCUSSION THREAD

**Discussion by leesi041** [community/community#81300] [Programming Help]
### Body

![스크린샷(29)](https://github.com/community/community/assets/154495071/b41c2944-9017-4fde-8562-527485ac4e4e)
In the code, "input" is executed. But "output" was not executed
![스크린샷(30)](https://github.com/community/community/assets/154495071/79817b3d-79d1-4f25-8f3d-1dfe160e9297)
Afterwards, I received a message saying that powershell was not installed, so I installed it. However, the message “Terminal” continued to appear as shown in the photo above.

Could you please explain whether Powershell affects running C++ in the VScode environment and how to install Powershell?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by chanwoo040531** [comment]
I don't think PowerShell is the reason. It seems like the problem is trying to run the application from the debug console. Changing the `externalConsole` to `true` in `launch.json` should solve it.

`launch.json` 에서 `externalConsole`을 `true`로 변경해보세요 :)
